mppm/about.md

```diff
@@ -6,6 +6,9 @@ description: Overview of Multiplayer Play Mode
 
 Use Multiplayer Play Mode to test multiplayer functionality within the Unity Editor. You can simulate up to four players (the Main Editor and three Virtual Players) simultaneously on the same development device while using the same source assets on disk. Multiplayer Play Mode can help you create multiplayer development workflows that reduce project build times, run it locally, and test the server-client relationship.
 
+## Compatibility
```

Comment:
General question: does it render correctly when you don't have a blank line after a header/before a list/between paragraphs?

Reply:
I think the space is necessary but I was honestly just fitting the convention of the existing docs - I'll test it.